### DISCUSSION TITLE
Randomly Stops Communicating/Updating

### DISCUSSION THREAD

**Discussion by tsvmitch** [fuatakgun/eufy_security#897] [Q&A]
I have two Homebase2s that I needed to sync. Eufy cannot do this internally, so HA and this integration to the rescue. 
My problem I have had for the entire time I've used this is that it will randomly stop updating and show the armed status as unavailable. Once I go and reload the integration it will typically reconnect and update. 
Am I doing something wrong?
Is there a way to reload every night? 
Any other suggestions?

**Comment by RonnieRocket2** [comment]
I have the same issue?
